test/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
```

Comment:
When I try to run "rspec" from the main repo nothing happens.  I think these specs need to be transferred to the spec folder and the standard test folder generated by Rails can be deleted (since we are using rspec)